GraphBLAS/Source/Template/GB_atomics.h

```diff
@@ -62,7 +62,12 @@
 // OpenMP 4.0 or later) with the "update" clause.  For MS Visual Studio, the
 // "update" clause is removed since it supports OpenMP 2.0.
 
-#if ( _OPENMP >= 201307 )
+#if ! defined (GB_ATOMIC_UPDATE) || (GB_ATOMIC_UPDATE < 199810)
```

Comment:
I think line 65 should be

#if ! defined ( _OPENMP) || (_OPENMP < 199810)

Reply:
You're absolutely right. Probably mis-clicked when copying and pasted without checking...
Should be fixed now.